Sources/Auth/Services/Signer/CocoaSigner.swift

```diff
@@ -0,0 +1,32 @@
+import Foundation
+
+protocol CacaoSignatureVerifying {
+    func verifySignature(_ cacao: Cacao) throws
+}
+
+struct CacaoSigner: CacaoSignatureVerifying {
```

Comment:
The file name is `CocoaSigner` while the type is `CacaoSigner`

Reply:
thanks